plugins/kn-migration/README.md

```diff
@@ -0,0 +1,66 @@
+# kn migration migrate
+
+Migrate Knative services from source cluster to destination cluster
+
+## Plugin introduction
+### Synopsis
+
+Migrate Knative services from source cluster to destination cluster
+
+```
+> kn migration --help
+Migrate Knative services from source cluster to destination cluster
+
+Usage:
+  migration [command]
+
+Available Commands:
+  help        Help about any command
+  list        List all knative resources
+  migrate     Migrate Knative services from source cluster to destination cluster
+```
+
+### Examples
+```
+  # Migrate Knative services from source cluster to destination cluster by export KUBECONFIG and KUBECONFIG_DESTINATION as environment variables
+  kn migration migrate --namespace default --destination-namespace default
+
+  # Migrate Knative services from source cluster to destination cluster by set kubeconfig as parameters
+  kn migration migrate --namespace default --destination-namespace default --kubeconfig $HOME/.kube/config/source-cluster-config.yml --destination-kubeconfig $HOME/.kube/config/destination-cluster-config.yml
+
+  # Migrate Knative services from source cluster to destination cluster and force replace the service if exists in destination cluster
+  kn migration migrate --namespace default --destination-namespace default --force
+
+  # Migrate Knative services from source cluster to destination cluster and delete the service in source cluster
+  kn migration migrate --namespace default --destination-namespace default --force --delete
+```
+
+### Options
+```
+      --delete                          Delete all Knative resources after kn-migration from source cluster
+      --destination-kubeconfig string   The kubeconfig of the destination Knative resources (default is KUBECONFIG_DESTINATION from environment variable)
+      --destination-namespace string    The namespace of the destination Knative resources
+      --force                           Migrate service forcefully, replaces existing service if any.
+  -h, --help                            help for migrate
+  -n, --namespace string                The namespace of the source Knative resources (default "default")
+```
+
+### Options inherited from parent commands
+```
+      --config string       kn config file (default is $HOME/.kn/config.yaml)
```

Comment:
Pay attention to the new fashion for kn's config location

Reply:
fixed in the second commit: https://github.com/knative/client-contrib/pull/6/commits/e77b98b6cebdccc7a81fa81f9c508488611a2d3b